alpine-nix-rails-nginx/config/nginx/static-content.conf

```diff
@@ -0,0 +1,39 @@
+# Default static content to be cached for 1 day
+expires 1d;
+
+if ($request_method !~ ^(GET|HEAD|OPTIONS)$) {
+  more_set_headers 'Allow: GET,HEAD,OPTIONS';
+  return 405;
+}
+
+if ($request_method = OPTIONS) {
+  more_set_headers 'Allow: GET,HEAD,OPTIONS';
+  return 204;
+}
+
+# Prefer the static files before proxying to the app
+try_files $uri @app;
+
+# Redirect requests for favicon to new icon
+location = /favicon.ico {
+  access_log off;
+  expires off;  # use default browser cache policy for 301
+  return 301 /assets/icons/favicon.ico;
```

Comment:
@mbj do you remember why we were doing this? I wonder if I should simply map the `favicon.ico` directly without a redirect.